src/com/mellanox/cicd/Matrix.groovy

```diff
@@ -837,6 +837,11 @@ def runK8(image, branchName, config, axis, steps=config.steps) {
 spec:
   containers:
     - name: ${cname}
+      env:
```

Comment:
Do we want this also in docker build function build_docker_on_k8 (line 1300) ?

Reply:
Agree, added to build_docker_on_k8